sr/robot3/robot.py

```diff
@@ -169,7 +170,9 @@ def is_arduino(cls, port: ListPortInfo) -> bool:
 
     def _init_metadata(self) -> None:
         """Fetch metadata from Astoria."""
-        self._metadata, self._code_path = GetMetadataConsumer.get_metadata()
+        (
+            self._metadata, self._code_path,
+        ) = GetMetadataConsumer.get_metadata(self._mqtt)
```

Comment:
```suggestion
        self._metadata, self._code_path = GetMetadataConsumer.get_metadata(
            self._mqtt,
        )
```

A linter isn't there to get in your way, it's there to help to remind you to make code readable. If you get up in a situation where you can't get the linter to pass in a readable way, it's better to add a per-line ignore.